homework-submission/todo-app-week1/src/components/ItemsComponent.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const ItemsComponent = ({ items, done, action, addItem, inputRef }) => {
+  let lis = [];
```

Comment:
You could find a better name for `lis`, also it should be const since you are not reassigning the array again but push or delete